solutions/standard/version.tf

```diff
@@ -0,0 +1,15 @@
+terraform {
+  required_version = ">= 1.3.0"
+
+  # Lock DA into an exact provider version - renovate automation will keep it updated
+  required_providers {
+    ibm = {
+      source  = "IBM-Cloud/ibm"
+      version = "1.66.0"
```

Comment:
This should be the latest version (1.67.1)... although renovate will fix this.

Reply:
sure, I have made this change.